meilisearch-release-changelog/v1.12.0.md

```diff
@@ -0,0 +1,137 @@
+
+Meilisearch v1.12 introduces significant indexing speed improvements, almost halving the time required to index large datasets. This release also introduces new settings to customize and potentially further increase indexing speed.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features. Consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and updates 🔥
+
+## Improve indexing speed
+
+Indexing time for huge dataset import (multiple millions of documents) is divided by two!
+
+TBD: give more data and metrics about improvements
+
+Done by @dureuill, @ManyTheFish, and @Kerollmops in #4900.
+
+## New index settings: use `facetSearch` and `prefixSearch` to improve indexing speed
+
+v1.12 introduces two new index settings: `facetSearch` and `prefixSearch`.
+
+Both settings allow you to skip parts of the indexing process. This leads to significant improvements to indexing speed, but may negatively impact search experience in some use cases.
+
+Done by @ManyTheFish in #5091
+
+### `facetSearch`
+
+Use this setting to toggle [facet search](https://www.meilisearch.com/docs/learn/filtering_and_sorting/search_with_facet_filters#searching-facet-values):
+
+```bash
+curl \
+  -X PUT 'http://localhost:7700/indexes/books/settings/facet-search' \
+  -H 'Content-Type: application/json' \
+  --data-binary 'true'
+```
+
+The default value for `facetSearch` is `true`. When set to `false`, this setting disables facet search for all filterable attributes in an index.
+
+### `prefixSearch`
+
+Use this setting to configure the ability to [search a word by prefix](https://www.meilisearch.com/docs/learn/engine/prefix) on an index:
+
+```bash
+curl \
+  -X PUT 'http://localhost:7700/indexes/books/settings/prefix-search' \
+  -H 'Content-Type: application/json' \
+  --data-binary 'disabled'
+```
+
+`prefixSearch` accepts one of the following values:
+
+- `"indexingTime"`: enables prefix processing during indexing. This is the default Meilisearch behavior
+- `"disabled"`: deactivates prefix search completely
+
+
+Disabling prefix search means the query `he` will no longer match the word `hello`. This may significantly impact search result relevancy, but speeds up the indexing process.
+
+## New API route: `/batches`
+
+The new `/batches` endpoint allow you to query information about task batches.
```

Comment:
```suggestion

To give more visibility into how tasks are processed and batched together, Meilisearch introduced two new routes that give a view of the batches.
These views will give you some stats on what was processed in the batch and also a detailed view of the progress Meilisearch made on the batch. Finally, an explanation of what happens when a task is marked as processing :tada: 
```

Reply:
In general I think we should aim to be as brief as possible in the changelog, so users can skim it and quickly assess if anything is important to them. 

Contextualising changes is more the work of the documentation. And contextualising changes in a fun way is more of the work of the blog, because documentarians hate fun.